client/src/App.tsx

```diff
@@ -82,22 +100,44 @@ function App() {
 	const [figures, setFigures] = useState<IFigure[]>([]);
 
 	const sounds: ISound = {
+		// Instrument
 		drum: drumBeat,
 		cymbal: cymbalHit,
 		guitar: guitarStrum,
-		meme: gotEm
+		trumpet: trumpet,
+		gong: gong,
+		harp: harp,
+		// Funny
+		meme: gotEm,
+		noice: noice,
+		stop_it: stop_it_get_some_help,
+		ahh: ahh,
+		air: air,
+		applause: applause,
+		groan: groan,
+		clang: clang,
+		horn: horn,
+		laugh: laugh,
+		// Nature
+		bee: bee,
+		dog: dog,
+		flying_fox: flying_fox,
+		lightning: lightning,
+		nature: nature,
+		sealion: sealion,
+		water: water
 	};
 
 	const playEmoji = useCallback((type: string) => {
 		const { x, y } = generateRandomXY();
 
-		setEmojis((emojis) =>
+		setEmojis(emojis =>
```

Comment:
please run `npm run prettier` before pushing code to keep code consistent. The reason is that we can focus only on important code changes and not stuff like spaces, parantheses.